vm/src/stdlib/array.rs

```diff
@@ -417,6 +418,9 @@ def_array_enum!(
     (Double, f64, 'd', "d"),
 );
 
+#[derive(Copy, Clone, Ord, PartialOrd, Eq, PartialEq, Debug)]
+pub struct PyUnicode(libc::wchar_t);
```

Comment:
this is confusign name because `PyUnicode` indicates `unicode` in Python2 which is `str` in Python3.
This is just wchar. So `WideChar` will fit enough which as referred in CPython codes.

Reply:
I got `PyUnicode` from [python3.8's doc](https://docs.python.org/3.8/library/array.html), where the C type is named `Py_UNICODE`. However, `Py_UNICODE` is deprecated and replaced by `wchar_t` in [python3.9](https://docs.python.org/3.9/library/array.html), so I agree with you to use `WideChar` making it clearer.